time.md

```diff
@@ -783,7 +782,7 @@ This is just an exercise in copying our `CLI` tests but with less dependencies
 func TestGame_Start(t *testing.T) {
```

Comment:
This section says "This is just an exercise in copying our CLI tests but with less dependencies". For me, it involved these changes: moving spies over to testing.go (putting them in the poker package), capitalizing and renaming the fields of SpyBlindAlerter (necessary once in a different package). I just want to make sure that those changes were expected at this point.